lib/routes/situations.js

```diff
@@ -39,4 +39,7 @@ module.exports = function(api) {
 
     api.route('/situations/:situationId/openfisca-request').get(allowShow, situations.openfiscaRequest);
 
+    api.route('/situations/:situationId/openfisca-test').get(situations.openfiscaTest);
+    api.route('/situations/:situationId/openfisca-test').post(situations.openfiscaTest);
```

Comment:
It would be more elegant to do [content-negotiation](https://fr.wikipedia.org/wiki/Négociation_de_contenu) based on the [requested `Content-Type`](http://expressjs.com/fr/api.html#res.format) rather than creating new endpoints, unless they expose different data.